README.md

```diff
@@ -27,7 +27,7 @@ This repo implements a new three-stage build system for Ember apps:
    and 3 improves our ability to innovate and experiment with taking the best
    parts of wider JS ecosystem tooling.
 
-You can read more about the motivation and key ideas in the [intro to the SPEC](SPEC.md).
+You can read more about the motivation and key ideas in the [intro to the SPEC](docs/spec.md).
```

Comment:
also referenced in https://github.com/embroider-build/embroider/blob/3a232cc04a68f86d0acfa5e7edfa764b990efdbf/packages/macros/README.md?plain=1#L9